go/ct/rlz/conditions_test.go

```diff
@@ -199,6 +199,58 @@ func TestCondition_CheckStorageConfiguration(t *testing.T) {
 	}
 }
 
+func TestCondition_CheckForExistingAccount(t *testing.T) {
+	state := st.NewState(st.NewCode([]byte{}))
+	state.Pc = 42
+	condition := AccountExists(Pc())
```

Comment:
I am very surprised to see the program counter being used as an address.

Reply:
Anything CT expression that returns a `U256` is ok in this place. I copied this from the `IsWarmChecks` but you are right, semantically this odd; updated to consume a stack parameter.